docs/definition.rst

```diff
@@ -1,27 +1,94 @@
 Execution Environment Definition
 ================================
 
-The execution environment (EE) definition file supports multiple versions.
+You define the content of your Execution Environment in a YAML file. By default, this file is called ``execution_environment.yml``. This file tells Ansible Builder how to create the Containerfile and build your container image. The Execution Environment definition file accepts four top-level sections: version, images, dependencies, and (optionally) additional_build_steps.
 
-  * Version 1: Supported by all ``ansible-builder`` versions.
-  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and later.
-  * Version 3: Supported by ``ansible-builder`` versions ``3.0`` and later.
+.. contents::
+   :local:
 
-If the EE file does not specify a version, version 1 will be assumed.
+version
+-------
+The schema version for Ansible Builder.
+Required.
+
+Example:
+
+.. code-block:: yaml
+
+   version: 3
 
 .. note::
+   If you are running an older version of Ansible Builder, you may need an older schema version. Please consult older versions of the docs for more information. We recommend using version 3, which offers substantially more configurability and functionality than previous versions.
+
+If the EE file does not specify a version, version 1 will be assumed. (IS THIS STILL TRUE???)
```

Comment:
@nitzmahone  can you confirm ^^ ?

Reply:
Yes, version 1 is assumed if missing. Not sure why that's in question.